gulpfile.babel.js

```diff
@@ -161,8 +161,7 @@ gulp.task('html', () => {
 });
 
 // Clean output directory
-gulp.task('clean', cb => del(['.tmp', 'dist/*', '!dist/.git'], {dot: true},
-  cb));
+gulp.task('clean', () => del(['.tmp', 'dist/*', '!dist/.git'], {dot: true}));
```

Comment:
Unrelated to this PR, but thought I would mention that the `dot` option isn't necessary here as explicit dot files/folders are always included. - Author of `del`. That means you could probably remove `'!dist/.git'` as it shouldn't match dotfiles when using globstar matching (needs testing though just to be sure).